packages/hardhat/contracts/interfaces/IMetadata.sol

```diff
@@ -3,6 +3,4 @@ pragma solidity ^0.8.6;
 
 interface IMetadata {
     function tokenURI(uint256 _tokenId) external view returns (string memory);
-
-    function isMetaDataFrozen() external view returns (bool);
```

Comment:
not sure i follow why we're removing this.

Reply:
because I added this wen I assumed we don't need the metadataFreeze in the auction contract but since the freeze vars in the metadata contract & auction contract are different so we don't need this function